react-native/react/common-adapters/button.js.flow

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+
+import React, {Component} from '../base-react'
+
+type Props = {
+  label: string,
+  style: Object,
+  primary: bool
```

Comment:
This should be boolean, but maybe bool also works? (http://flowtype.org/docs/quick-reference.html#built-in-primitive-types)